proto/stat_var.proto

```diff
@@ -126,11 +127,12 @@ message StatVarGroupNode {
     // Name suitable for search.
     string search_name = 2;
     // Names suitable for search.
-    repeated string search_names = 5;
+    repeated string search_names = 4;
     // Name suitable for display in tree.
     string display_name = 3;
+    //------------------ Below are fields not in original cache.
```

Comment:
... and thus we start with a large tag number

Reply:
Done